src/js/components/FootNav.js

```diff
@@ -10,8 +10,8 @@ class FootNav extends React.PureComponent {
     info: PropTypes.object.isRequired,
     type: PropTypes.string,
     nav: PropTypes.shape({
-      previous: PropTypes.number,
-      next: PropTypes.number,
+      previous: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+      next: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
If the component seems small enough, you may consider rewriting it in Typescript.

Reply:
Yeah, I don't have much experience in typescript. I will look it up and will ping you about this for doubts.

Reply:
Merging to dev for now.